### PR TITLE
[FIX] Correct PDw suffix description

### LINKED ISSUE #1572: Mistake in the PDw suffix description
I stumbled on what appears to be a mistake in the [PDw suffix description](https://github.com/bids-standard/bids-specification/blob/ada2c5d445f1c0eab306e7a0d691e1889f1ffe4c/src/schema/objects/suffixes.yaml#L237). It says:

> In spin-echo sequences this contrast is achieved at short repetition and long echo times.

However, the opposite is true, i.e., [in spin-echo sequences, this contrast is achieved at long repetition and short echo times, just like in gradient echo sequences](https://mriquestions.com/image-contrast-trte.html). This description section should be changed to something like: "In both spin-echo and gradient-echo sequences, this contrast is achieved at long repetition and short echo times, and at small flip angles in gradient-echo."

### PR DESCRIPTION
**Pull request by jeremie-fouquet**
Fixes #1572.

### CODE REVIEW

**Review by effigies — APPROVED** [review]
This looks right to me. @agahkarakuzu Could I bug you for a second opinion?

**Comment by jhlegarreta** [comment]
I'm not an expert on sequences/contrast generation, but looks good to me.

**Comment by effigies** [comment]
@ericearl https://radiopaedia.org/articles/proton-density-weighted-spin-echo-images-1

**Comment by ericearl** [comment]
@effigies and @jeremie-fouquet I forgot to tag you. See suggestion/comment above.

**Comment by effigies** [comment]
@ericearl You may have a review draft comment that didn't get posted.

**Review by ericearl — CHANGES_REQUESTED** [review]
@effigies Did this post the changes?

File: src/schema/objects/suffixes.yaml
Comment (effigies) [reply src/schema/objects/suffixes.yaml:234]: Looks good to me. WDYT @jeremie-fouquet?

**Review by sappelhoff — APPROVED** [review]
Given that there is some contention (https://github.com/bids-standard/bids-specification/pull/1578#discussion_r1304742957) about the additions, I suggest that **given the additions being an add-on**, we merge this PR, and potentially (if @ericearl wishes) open the **add-on** as a new PR, as also suggested my @jeremie-fouquet in https://github.com/bids-standard/bids-specification/pull/1578#discussion_r1320452068

**Comment by sappelhoff** [comment]
re-requested your review @effigies so that you can make an assessment of my comment (https://github.com/bids-standard/bids-specification/pull/1578#pullrequestreview-1701206043) and directly merge if you deem appropriate. (With @ericearl currently being away)

File: src/schema/objects/suffixes.yaml
Comment (nicholst) [reply src/schema/objects/suffixes.yaml:234]: What about Eric's text, minus the absolute times, plus a mention that "; also, for gradient echo, this weighting is also possible with a short TR (TR<<T1) and a small flip angle."?

Comment (effigies) [reply src/schema/objects/suffixes.yaml:234]:
```suggestion
    This contrast is achieved at short echo times and long repetition times;
    for gradient echo, this weighting is also possible with a short TR (TR<<T1) and a small flip angle.
```

**Review by effigies — APPROVED** [review]
Applied Tom's suggestion. @ericearl @jeremie-fouquet please review and let us know what you think. In the absence of an objection, we will merge on Monday (Nov 6).

**Review by jeremie-fouquet — COMMENTED** [review]
Looks correct to me. Thank you.

**Comment by effigies** [comment]
Alright, let's get this in. Thank you for your patience!

**Comment by effigies** [comment]
@jeremie-fouquet please be sure to add/update your information in https://github.com/bids-standard/bids-specification/wiki/Recent-Contributors!

**Pull request merged**
